components/eventing-controller/controllers/subscriptionv2/eventmesh/test/utils.go

```diff
@@ -400,11 +401,19 @@ func countEventMeshRequests(subscriptionName, eventType string) (int, int, int)
 }
 
 func getEventMeshSubFromMock(subscriptionName, subscriptionNamespace string) *eventMeshtypes.Subscription {
-	nm1 := emTestEnsemble.nameMapper.MapSubscriptionName(subscriptionName, subscriptionNamespace)
-	key := fmt.Sprintf("%s/%s", "/messaging/events/subscriptions", nm1)
+	key := getEventMeshSubKeyForMock(subscriptionName, subscriptionNamespace)
 	return emTestEnsemble.eventMeshMock.Subscriptions.GetSubscription(key)
 }
 
+func getEventMeshSubKeyForMock(subscriptionName, subscriptionNamespace string) string {
+	nm1 := emTestEnsemble.nameMapper.MapSubscriptionName(subscriptionName, subscriptionNamespace)
+	return fmt.Sprintf("%s/%s", "/messaging/events/subscriptions", nm1)
+}
+
+func getEventMeshKeyForMock(name string) string {
+	return fmt.Sprintf("%s/%s", "/messaging/events/subscriptions", name)
+}
```

Comment:
If using the same value "/messaging/events/subscriptions" is intentional, please use a shared const.

Reply:
updated